<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>745</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1022</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ema2wav</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>EMA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <widget class="QListView" name="ema_files_view"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="ema_directory_line_edit"/>
         </item>
         <item>
          <widget class="QPushButton" name="open_ema_dir_btn">
           <property name="text">
            <string>open EMA directory</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>EMA information</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>EMA device:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ema_device_info">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>EMA sample rate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ema_samplerate_info">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>EMA channels:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ema_channel_info">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>261</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>WAVE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5"/>
       </item>
       <item>
        <widget class="QListView" name="wave_files_view"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="wave_directory_line_edit"/>
         </item>
         <item>
          <widget class="QPushButton" name="open_wave_dir_btn">
           <property name="text">
            <string>open WAVE directory</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>WAVE information</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>WAVE sample rate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="wave_samplerate_info">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>WAVE channels:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="wave_channel_info">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>211</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Channel allocation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="channel_table"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="add_channel_btn">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_channel_btn">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>221</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Parameters of interest</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="parameter_table"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QPushButton" name="add_parameter_btn">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_parameter_btn">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>441</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>EMA filtering</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QRadioButton" name="no_filter_radio_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>none</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QRadioButton" name="moving_average_btn">
         <property name="text">
          <string>moving average</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="moving_average_window_input">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>window size</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QRadioButton" name="butter_lp_btn">
         <property name="text">
          <string>Butterworth lowpass filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="butter_lp_cutoff_input">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>cutoff</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="butter_lp_order_input">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>order</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>1001</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>CONFIG path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="config_dir_line_input"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="load_CONFIG_btn">
         <property name="text">
          <string>load CONFIG</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>output directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="output_dir_line_input"/>
         </item>
         <item>
          <widget class="QPushButton" name="open_output_directory_btn">
           <property name="text">
            <string>open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <item>
          <widget class="QCheckBox" name="export_audio_ema_checkbox">
           <property name="text">
            <string>export audio+EMA</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="export_csv_checkbox">
           <property name="text">
            <string>export to CSV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="export_raw_ema_checkbox">
           <property name="text">
            <string>export raw EMA</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="start_conversion_btn">
         <property name="text">
          <string>convert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
